README.md

```diff
@@ -91,11 +92,13 @@ maven-public      maven2   group    http://localhost:8081/repository/maven-publi
 ```
 
 Create a repository:
+
 ```bash
-$ nexus3 repository create hosted raw reponame
```

Comment:
👍 heh, I always groan when I want to copy/paste an example command but then need to remove the `$`.